### PR TITLE
Added dream team exercise

### PR DESCRIPTION
**Pull request by stevebrownlee**
This is an exercise that has students practices making many classes, creating instances of them, adding them to one, of two, Lists, and then iterating over the Lists.

### CODE REVIEW

File: orientation/exercises/12_DREAM_TEAM.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Once you're done, you should have 6 different types in total, each with the properties and methods above.
+
+
+1. Create two groups (i.e. `List`) that will hold three teammates each. These two lists represent one team that will be the server side team, and one that will be the client side team.
+1. Instantiate one instance of each of your teammates.
+1. Put your teammates into the appropriate team.
+    ```cs
+    serverSide.Add(Ed);
+    serverSide.Add(Eliza);
+    serverSide.Add(Kathleen);
```
Comment (megducharme) [inline orientation/exercises/12_DREAM_TEAM.md]: she goes by Kathy :)

**Comment by megducharme** [comment]
Maybe this is where you can intro overloading and overriding methods? Or maybe throw in an interface to make it a little more complicated.

**Comment by megducharme** [comment]
LGTM! Lets do it!

**Comment by hannahhall** [comment]
👍

**Pull request merged**